cpu.go

```diff
@@ -17,8 +17,8 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-const measureInterval = time.Second * 10
-const cpuGetUtilisationTimeout = time.Second * 10
+const measureInterval = time.Second * 20
```

Comment:
We can increase the timeouts but I don't think this will change anything with our current issue. I would guess it's something "deeper"